content/guides/references/configuration.md

```diff
@@ -145,177 +177,161 @@ file.
-```
+:::cypress-config-example{noJson}
 
-```shell
-cypress run --browser firefox --config viewportWidth=1280,viewportHeight=720
+```js
+{
+  e2e: {
+    // e2e options here
+  }
+}
 ```
 
-For more complex configuration objects, you may want to consider passing a
-[JSON.stringified](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify)
-object surrounded by single quotes.
-
-```shell
-cypress open --config '{"watchForFileChanges":false,"testFiles":["**/*.js","**/*.ts"]}'
-```
+:::
 
-### Runner Specific Overrides
+### component
 
-You can override configuration for either the E2E or
-[Component Testing](/guides/component-testing/introduction/) runner using the
-`e2e` and `component` options.
+These options are available to be specified inside the `component` configuration
+object:
 
-#### Examples
+| Option               | Default                                  | Description                                                                                                                                                                                         |
+| -------------------- | ---------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `devServer`          | `null`                                   | Required function used to configure the component testing dev server. [Please read the notes for examples on using this.](#devServer-devServerConfig)                                               |
+| `devServerConfig`    | `null`                                   | Optional dev server configuration. Config options will be determined by the dev server. [Please read the notes for examples on using this.](#devServer-devServerConfig)                             |
+| `setupNodeEvents`    | `null`                                   | Function in which node events can be registered and config can be modified. Takes the place of the (deprecated) plugins file. [Please read the notes for examples on using this.](#setupNodeEvents) |
+| `supportFile`        | `cypress/support/index.js`               | Path to file to load before test files load. This file is compiled and bundled. (Pass `false` to disable)                                                                                           |
+| `specPattern`        | `**/*.cy.{js,jsx,ts,tsx}`                | A String or Array of glob patterns of the test files to load. <br><br>Note that any files found matching the `e2e.specPattern` value will be automatically **excluded.**                            |
+| `excludeSpecPattern` | `['/snapshots/*', '/image_snapshots/*']` | A String or Array of glob patterns used to ignore test files that would otherwise be shown in your list of tests. [Please read the notes on using this.](#excludeSpecPattern)                       |
```

Comment:
```suggestion
| `excludeSpecPattern` | `['/snapshots/*', '/image_snapshots/*']` | A String or Array of glob patterns used to ignore spec files that would otherwise be shown in your list of specs. [Please read the notes on using this.](#excludeSpecPattern)                       |
```

Reply:
[ce4d48c](https://github.com/cypress-io/cypress-documentation/commit/ce4d48cee2d5ea71ae9be4ac11f679e919987999)